examples/buildkit0/buildkit.go

```diff
@@ -17,8 +17,8 @@ type buildOpt struct {
 func main() {
 	var opt buildOpt
 	flag.BoolVar(&opt.withContainerd, "with-containerd", true, "enable containerd worker")
-	flag.StringVar(&opt.containerd, "containerd", "v1.2.0-rc.1", "containerd version")
-	flag.StringVar(&opt.runc, "runc", "a00bf0190895aa465a5fbed0268888e2c8ddfe85", "runc version")
+	flag.StringVar(&opt.containerd, "containerd", "v1.2.1", "containerd version")
+	flag.StringVar(&opt.runc, "runc", "v1.0.0-rc6", "runc version")
```

Comment:
@AkihiroSuda should we use the same version of runc as is used in containerd v1.2.1? see https://github.com/containerd/containerd/blob/v1.2.1/vendor.conf#L23

That version was updated to fix a regression in runc; https://github.com/containerd/containerd/commit/4c327b8e5628d7f296b4a6db8ca8da8c7a4dd367

Reply:
Yes, but the regression does not seem to affect on BuildKit.